actions/tests/actions_tests/test_actions_custom_data_models/test_table_result.yml

```diff
@@ -3,6 +3,8 @@ result:
   columns:
   - a
   - b
+  description: null
```

Comment:
This actually means we broke backward compatibility because additional fields are available in the Table model_dump (so, we should update to 2.0 and note that in the changelog accordingly).